README.md

```diff
@@ -20,3 +21,27 @@ randomly without memory instead of systematically trying them?
 
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
+
+In my code I am getting all the permutations and checking each one if it is sorted.
+We have N which is the length of the array. In the code there are 4 sections to anaylze.
+swapElements: This function swaps the elements in the array in order to create the permutations.
+isSorted: Checks if the array is sorted.
+generatePermutations: This function uses swapElements to genereate the permutations of the array.
+permutationSort: Uses generated permutations to find the sorted array. 
+
+## Worst Case 
+In the worst case senerio the array would be in reverse order causing the algorithm to need to create and check every permutation before finding the sorted array. If the array is size N the number of permutations would be N! making the worst case time complexity $O(N!)$
```

Comment:
What do you need to do for each permutation?

Reply:
would this be to check if the permutation is sorted???